systems/node/state/pb/state.proto

```diff
@@ -0,0 +1,114 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * Copyright (c) 2023-present, Ukama Inc.
+ */
+syntax = "proto3";
+option go_package = "github.com/ukama/ukama/systems/node/state/pb/gen";
+package ukama.node.state.v1;
+import "validator.proto";
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/wrappers.proto";
+import "ukama/node.proto";
+
+service StateService {
+    rpc AddState (AddStateRequest) returns (AddStateResponse);
+    rpc GetStateById (GetStateByIdRequest) returns (GetStateByIdResponse);
+    rpc GetStates(GetStatesRequest) returns(GetStatesResponse);
+    rpc GetLatestState(GetLatestStateRequest) returns (GetLatestStateResponse);  
+    rpc UpdateState(UpdateStateRequest) returns (UpdateStateResponse);
```

Comment:
We need to get Node State History (could be based on a time filter or you could be just last N number of states)